daemon/cmd/daemon_main.go

```diff
@@ -185,6 +185,9 @@ func InitGlobalFlags(cmd *cobra.Command, vp *viper.Viper) {
 	flags.Bool(option.AutoCreateCiliumNodeResource, defaults.AutoCreateCiliumNodeResource, "Automatically create CiliumNode resource for own node on startup")
 	option.BindEnv(vp, option.AutoCreateCiliumNodeResource)
 
+	flags.StringSlice(option.ExcludeNodeLabelPatterns, []string{}, "List of k8s node label regex patterns to be excluded from CiliumNode")
+	option.BindEnv(vp, option.ExcludeNodeLabelPatterns)
```

Comment:
Nodes are additionally synchronized in the kvstore by the operator [here](https://github.com/cilium/cilium/blob/main/operator/cmd/cilium_node.go#L164-L167). My understanding is that we should additionally register this flag in the operator as well, to ensure consistency if the labels are filtered.

Reply:
This is addressed now.